packages/python/plotly/plotly/__init__.py

```diff
@@ -102,7 +102,8 @@ def plot(data_frame, kind, **kwargs):
     if kind == "line":
         return line(data_frame, **kwargs)
     if kind == "area":
-        return area(data_frame, **kwargs)
+        new_kwargs = {k: kwargs[k] for k in kwargs if k not in ["stacked"]}
```

Comment:
Maybe a question for @nicolaskruchten as much as for you @kaibir - since there are already other examples of these exclusions in other `kind`s, but how do we know what needs to be excluded?

Reply:
It raised in a test 🙈